build/Publish.targets

```diff
@@ -175,6 +189,9 @@
                     AccountKey="$(PublishBlobFeedKey)"
                     ItemsToPush="@(PackagesToPublishToTransport)"
                     Overwrite="$(PushToBlobFeed_Overwrite)"
+                    ManifestBuildId="$(BuildNumber)"
+                    ManifestCommit="$(CommitHash)"
+                    ManifestName="aspnet"
```

Comment:
Oh, if you have a property laying around with the branch name in it, it would be good to pass that to `ManifestBranch`. It is optional, though.

Reply:
We pass in an environment variable, but I'd rather not use it. I think the commit sha does a much better job at it and we're guaranteed to include it.

Reply:
+1 - I'd prefer not to include branch name.

Reply:
That's ok with me.